runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -4447,7 +4450,10 @@ private void doEncodeConnack(
             String16FW reason,
             int version)
         {
-
+            if (reasonCode == SUCCESS)
+            {
+                event.clientConnected(traceId, routedId, guard, authorization, clientId.asString());
+            }
```

Comment:
```suggestion
            if (reasonCode == SUCCESS)
            {
                events.onClientConnected(traceId, routedId, guard, authorization, clientId.asString());
            }

```
`events` not `event`
blank line to improve readability
`onXxx` method naming for event triggers